splinepy/_spline.py

```diff
@@ -1,14 +1,11 @@
-import abc
-import logging
 import copy
 import os
-import itertools
 
 import numpy as np
 
 from splinepy import utils
 from splinepy import io
-from splinepy import _splinepy as core
```

Comment:
What happened to this line? does this work?

Reply:
whopsie 🙈